charts/cp-schema-registry/values.yaml

```
@@ -83,6 +83,39 @@ resources: {}
 ## Custom pod annotations
 podAnnotations:
   cluster-autoscaler.kubernetes.io/safe-to-evict: "true"  # Allow the autoscaler to downscale the cluster.
+  sidecar.istio.io/inject: "false"
+  ad.datadoghq.com/{{ .Release.Name }}-server.logs: '[{"source":"schema-registry","service":"{{ .Release.Name }}"}]'
```

According to Datadog'sdocs (https://docs.datadoghq.com/agent/kubernetes/integrations/?tab=kubernetes#configuration) and our other integrations, the annotation must be called:
```suggestion
  ad.datadoghq.com/[containerName].logs: '[{"source":"schema-registry","service":"{{ .Release.Name }}"}]'
```
_NB: this is not a real suggestion, just to explain my point._

So, I'm not sure that `{{ .Release.Name }}-server` is the right configuration here.

---

My response above (with links to other charts) was really not suited here, sorry. You can just forget it 😝 

So, in my suggestion above, I wrote `[containerName]`: this was NOT a real syntax to use.
I wanted to explain that the annotation must contain the name of the *container*.

I just dig a little bit: the container's name is defined here: https://github.com/blablacar/cp-helm-charts/blob/master/charts/cp-schema-registry/templates/deployment.yaml#L62
And it is using this function: https://github.com/blablacar/cp-helm-charts/blob/master/charts/cp-schema-registry/templates/_helpers.tpl#L5-L7
BTW: this is weird have a *dynamic* name for a container, but since it's a fork of an upstream chart, let's keep it.

So, in the end, my suggestion is the use exactly the same function that in Deployment (because datadog needs exactly the name of the container):
```suggestion
  ad.datadoghq.com/{{ template "cp-schema-registry.name" . }}-server.logs: '[{"source":"schema-registry","service":"{{ .Release.Name }}"}]'
```

---

since the container name doesn't depend on the release name we could just use the container name directly like in the cassandra chart

---

should be ok now, it's like for the cassandra chart, the container name is hardcoded